packages/captp/src/captp.js

```diff
@@ -167,8 +208,11 @@ export const makeCapTP = (
         // and use that to construct the slot name.  Promise slots are prefaced
         // with 'p+'.
         lastPromiseID += 1;
-        const promiseID = lastPromiseID;
-        slot = `p+${promiseID}`;
+        slot = `p+${lastPromiseID}`;
+        const promiseID = reverseSlot(slot);
+        if (exportHook) {
+          exportHook(val, slot);
```

Comment:
Yes, I can see how this would be very useful. But why no `importHook`? I'm happy to omit it but would like to understand the asymmetry.